meta-xt-control-domain/recipes-guest/doma/files/doma-restart-monitor.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Android VM restart service
+
+[Service]
+Type=simple
+
+ExecStart=/usr/lib/xen/bin/doma-restart-monitor-ExecStart.sh
+
+ExecStopPost=echo "DomA restart monitor stopped!"
+
+Restart=on-failure
+RemainAfterExit=true
```

Comment:
> RemainAfterExit=
>Takes a boolean value that specifies whether the service shall be considered active even when all its processes exited.

Is this a correct option for this service?

Reply:
Hi @lorc, I think it is not needed here, as the script of this service remains active until doma restart.

Accepted.

Reply:
Hi @lorc,

Done for doma-restart-monitor.service and domu-restart-monitor.service